pages/services/kafka-zookeeper/2.6.0-3.4.14/security/index.md

```diff
@@ -0,0 +1,299 @@
+```
+#include /services/include/security-kerberos-ad.tmpl
+
+#include /services/include/security-service-keytab.tmpl
+
+#### Install the Service
+
+Install the DC/OS {{ model.techName }} service with the following options in addition to your own:
+```json
+{
+    "service": {
+        "security": {
+            "kerberos": {
+                "enabled": true,
+                "kdc": {
+                    "hostname": "<kdc host>",
+                    "port": <kdc port>
+                },
+                "primary": "<service primary default zookeeper>",
+                "realm": "<realm>",
+                "keytab_secret": "<path to keytab secret>",
+                "debug": <true|false default false>
+            }
+        }
+    }
+}
+```
+
+<p class="message--note"><strong>NOTE: </strong>It is possible to enable Kerberos after initial installation but the service may be unavailable during the transition. Additionally, your ZooKeeper clients will need to be reconfigured. For more information see the following section.</p>
+
+#### Enabling Kerberos After Deployment
+
+It is possible to enable Kerberos authentication after the deployment of DC/OS {{ model.techName }}. As described in the (Rolling Upgrade)[https://cwiki.apache.org/confluence/display/ZOOKEEPER/Server-Server+mutual+authentication] section of the Apache ZooKeeper documentation, this requires multiple rolling restarts of the ZooKeeper ensemble and client connectivity may be lost at times.
+
+Assuming that DC/OS {{ model.techName }} was initially deployed with `service.security.kerberos.enabled` set to `false`, the following steps can be used to enable Kerberos for the service.
+
+Firstly -- assuming the same Kerberos settings as discussed in [Configure Kerberos Authentication](#configure-kerberos-authentication) -- create the keytab for the Kerberos principals and add this keytab to the DC/OS Secret Store as described in the [Create principals](#create-principals) and [Place Service Keytab in DC/OS Secret Store](#place-service-keytab-in-dc-os-secret-store) sections. Then create a `kerberos-toggle-step-1.json` file with the following contents:
+
+```json
+{
+    "service": {
+        "security": {
+            "kerberos": {
+                "enabled": true,
+                "kdc": {
+                    "hostname": "<kdc host>",
+                    "port": <kdc port>
+                },
+                "primary": "<service primary default zookeeper>",
+                "realm": "<realm>",
+                "keytab_secret": "<path to keytab secret>",
+                "debug": <true|false default false>,
+                "advanced": {
+                    "required_for_quorum_learner": false,
+                    "required_for_quorum_server": false,
+                    "required_for_client": false
+                }
+            }
+        }
+    }
+}
+```
+where it is important to note the `service.security.kerberos.advanced` section that is present here.
```

Comment:
Use Important message format from https://wiki.mesosphere.com/display/DOCS/Documentation+Style+Guide#DocumentationStyleGuide-Notes  if needed 
OR 
Write `You can read more information from `service.security.kerberos.advanced` section`